src/admin/content-block/components/ContentBlockPreview/ContentBlockPreview.tsx

```diff
@@ -76,38 +76,19 @@ const ContentBlockPreview: FunctionComponent<ContentBlockPreviewProps> = ({
 		}
 	});
 
+	// TODO: Change BlockCTA to the way Buttons works so that we don't have to add navigate to each CTA element + then we can remove one of the two following conditional statements..
+	if (blockState.blockType === ContentBlockType.Buttons) {
+		stateToSpread.elements.forEach(({ action }: any) => {
+			stateToSpread.navigate = () => {
+				navigateToContentType(history, action);
```

Comment:
we hebben reeds een aantal navigate to url function in de client. Deze nemen allemaal history as 2de parameter.
Zou je het zien zitten om de volgorde van deze parameters om te draaien, zodat de functies overeen komen?

Reply:
Tuurlijk zie ik dat zitten. :)